epub2speech/tts/azure_provider.py

```
@@ -39,100 +37,35 @@ def convert_text_to_audio(
         self,
         text: str,
         output_path: Path,
-        voice: str | None = None
+        voice: str,
     ):
         if not self._speech_config:
             raise RuntimeError("Azure Speech not properly configured - cannot convert text to audio")
 
         if not text or not text.strip():
             raise ValueError("Empty text provided for conversion")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)